templates/zerver/help/mute-a-topic.md

```diff
@@ -1,43 +1,74 @@
 # Mute a topic
 
 If you would like to stop receiving notifications from a certain topic, you can
-choose to mute it.
+choose to mute it. Note that you can also [mute the whole stream](/help/mute-a-stream)
+as well.
 
-There are two main ways of muting a topic; you can mute a topic through the
-stream list or through your view.
+There are two main ways of muting a topic; you can mute the topic through the
+stream list or through your current filter.
 
-## Muting a topic through the stream list
+## Mute a topic through the stream list
 
-1. Find the **Streams** section located on the left sidebar. In the
-stream list, click on the stream that the topic you want to mute
-belongs to. Upon clicking on the stream, a dropdown with all the
-topics in the stream will appear.
+1. Find the **Streams** section located on the left sidebar. In the list, click
+on the stream that the topic you want to mute belongs to. Upon clicking on the
+stream's name, all the topics of the stream will appear.
 
-3. In the stream topics dropdown, hover over a topic to reveal a {!down-chevron.md!}
-To mute the topic, simply select the **Mute the topic (topic name)**
-option from the dropdown.
+2. In the stream's topics list, hover over a topic to reveal a {!down-chevron.md!}
+To mute the topic, simply select the **Mute the topic (topic name)** option
+from the dropdown.
 
-5. Upon clicking **Mute the topic (topic name)**, a tooltip titled **Topic Muted** will
-appear, and the topic will be grayed out in the stream list dropdown, confirming
-the success of your muting of the topic.
+3. Upon clicking **Mute the topic (topic name)**, a tooltip titled **Topic Muted**
+will appear, and the topic will be grayed out in the stream list dropdown,
+confirming the success of your muting of the topic.
 
-## Muting a topic through your view
+## Mute a topic through your current filter
```

Comment:
Just to make sure - is it the new style guide to use `filter` instead of `view`?